### PR TITLE
Namespacification

### PR DESCRIPTION
**Pull request by vincentpierre**
[Namespace created] Added the namespace MLAgents on the C# scripts

### CODE REVIEW

File: unity-environment/Assets/ML-Agents/Editor/MLAgentsEditModeTest.cs [hunk 1]
```diff
@@ -2,7 +2,7 @@
 using NUnit.Framework;
 using System.Reflection;
 
-namespace MLAgentsTests
+namespace MLAgents.MLAgentsTests
```
Comment (awjuliani) [inline unity-environment/Assets/ML-Agents/Editor/MLAgentsEditModeTest.cs]: Should this be be `MLAgents.Tests`?

Comment (vincentpierre) [reply unity-environment/Assets/ML-Agents/Editor/MLAgentsEditModeTest.cs]: I see what what you did there

**Comment by eshvk** [comment]
`develop` has  these files:
```
PyramidAcademy.cs
PyramidAgent.cs
PyramidArea.cs
PyramidSwitch.cs
```
which are not in your branch. Make sure you pull, rebase add the namespace there as well.

**Comment by awjuliani** [comment]
Perhaps we should use a finer-scale for namespacing. High-level `MLAgents`, and then `MLAgents.Examples` and `MLAgents.Tester`, etc for sub-folders. It doesn't make sense for example environments to use the high-level namespace, since they aren't actually essential parts of ML-Agents.

**Comment by vincentpierre** [comment]
The examples do not contribute to the namespace, they just use it. But your point is right for the scripts in shared assets.

**Pull request merged**